1.	Identify-Basic-Internet-Problems-with-ping.md

```diff
@@ -0,0 +1,113 @@
+![Sonar screen](https://media.licdn.com/mpr/mpr/jc/AAEAAQAAAAAAAAfoAAAAJDRmY2I4ZmE1LTk4ZmQtNGUzOS1hOWQ4LTNkMGJkOWQxNjNkYg.png)
+
```

Comment:
The image link is broken.I am getting broken image when page loads. And the source of the image shows this

![untitled4](https://cloud.githubusercontent.com/assets/5607371/14708694/186111be-07ea-11e6-82e9-324a7c11dce6.png)